src/spec.md

```diff
@@ -23,17 +23,26 @@ Messenger implementations expose the API through a `webxdc.js` module. To activa
 window.webxdc.sendUpdate(update, descr);
 ```
 
+Send a status update to all peers.
+
 - `update`: an object with the following properties:  
-    - `update.payload`: any javascript primitive, array or object.
+    - `update.payload`: string, number, boolean, array, object or `null`.
+       MUST NOT be `undefined`.
+       Binary data buffers as used in `File`, `Blob`, `Int*Array` etc. are skipped;
+       if needed, use eg. base64.
```

Comment:
optional nitpicking: maybe also mention that it must be "json serialisable"

Reply:
also thought about that, but that would mean ppl have to understand that part (which is also only an implementation detail). i think an explicit list is better